README.md

```diff
@@ -27,4 +27,4 @@ You mean besides incredibly good karma? Here are some very likely outcomes:
 - You’ll meet interesting new people outside your usual developer group.
 - You’ll probably learn something new by answering questions you never thought of asking yourself.
 - Eternal gratitude from your group of attendees and the Django Girls community.
-- You’ll help break down the stereotype that women are not interested in technology and programming and thus help your sisters, female friends and relatives, (future) daughters have more choices in their future. And you can add all that on your CV!
+- You’ll help break down the stereotype that women are not interested in technology and programming and thus help your sisters, female friends and relatives, and (future) daughters have more choices in their future. And you can add all that on your CV!
```

Comment:
Yes, but here you have "and" two times in one sentence. The proper one should be:
```
You’ll help break down the stereotype that women are not interested in technology and programming and thus help your sisters, female friends, relatives and (future) daughters have more choices in their future.```

I'm also not sure that you should place a comma before the and in the sentence.